lib/miq_automation_engine/service_models/miq_ae_service_network_router.rb

```diff
@@ -8,8 +8,5 @@ class MiqAeServiceNetworkRouter < MiqAeServiceModelBase
     expose :network_ports,         :association => true
     expose :vms,                   :association => true
     expose :private_networks,      :association => true
-
-    expose :raw_update_network_router
-    expose :raw_delete_network_router
```

Comment:
Did these methods ever exist in the active_record models?  They do not exist on the commit that introduced the service_model methods (https://github.com/ManageIQ/manageiq/pull/11477/commits/060254cf3cb21c10b44e52a011b8c75290f0a4d8) and the tests are only checking that the service model contains the method.

Also `git log -S "raw_update_network_router"` only references the commit above.  So I'm guessing these were never usable in the first place.  So ✂️ .